lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -651,7 +651,7 @@ void SessionStateEnforcer::prepare_response_to_access(
   rsp->mutable_session_ambr()->set_max_bandwidth_dl(
       config.rat_specific_context.m5gsm_session_context()
           .default_ambr()
-          .max_bandwidth_ul());
+          .max_bandwidth_dl());

```

Comment:
Was this a bug? Maybe we should have a unit test to cover these conversions?

Reply:
Sure.. Will add the testcase